queries.sql

```diff
@@ -15,4 +16,43 @@ SELECT * From animals Where neutered = true ;
 /* The 7th qeury*/
 SELECT * From animals Where name not in ('Gabumon'); 
 /* The 8th query*/
-SELECT * From animals Where (weight_kg >= '10.4' and weight_kg <= '17.3');
\ No newline at end of file
+SELECT * From animals Where (weight_kg >= '10.4' and weight_kg <= '17.3');
+
+/* Second Project*/
```

Comment:
- [ ] Please create a `query` that `UPDATE` the animal's table by setting the species column to unspecified, then roll back the change
```sql
 BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; `-- verify that change was undone`
```